### PR TITLE
Clean up base table styling

### LINKED ISSUE #390: Audit base tables
## Context

- Styling should only be applied to single element selectors. No need for `tbody th` when `th` will do
- Is the styling as lean as it can be?
- Does it use appropriate breakpoints?
- Styling should be mobile-first, max-width media queries should be avoided where possible.

## Audit Checklist

- [ ] Browser testing
- [ ] Accessibility testing
- [ ] Sass linting
- [ ] Update documentation
- [ ] Convert sizing units to rems
- [ ]  Ensure only elements are styled directly

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Simplify base table styling
- Reduce specificity caused by nested selectors

## QA

- Sanity check code
- QA base table pattern in vf.io

## Details

Fixes #390

### CODE REVIEW

File: scss/_base_tables.scss [hunk 1]
```diff
@@ -5,49 +5,31 @@
...
-    th[scope='col'] {
-      text-align: center;
-    }
-
-    thead th:first-of-type {
-      text-align: left;
-    }
-
+  th,
+  td {
```
Comment (anthonydillon) [inline scss/_base_tables.scss]: Why are you not needing these in table?

File: scss/_base_tables.scss [hunk 2]
```diff
@@ -5,49 +5,31 @@
...
 
-    table {
-      display: block;
-    }
+  thead th {
+    border-collapse: separate;
+    border-spacing: 0 .5rem;
+    background-color: $color-warm-grey;
+    color: $color-light-grey;
+    font-weight: normal;
```
Comment (anthonydillon) [inline scss/_base_tables.scss]: Can you use font weight 400 please.

**Comment by anthonydillon** [comment]
Thanks Barry. The tables have been redesigned to a clearer style. It would be great to capture that change in this PR. @yaili could you link to the designs or specs please?

**Comment by anthonydillon** [comment]
Only marking as code -1 as there is a small tweak request. The other content is a question.

**Comment by yaili** [comment]
@anthonydillon yes: https://github.com/ubuntudesign/vanilla-design/tree/master/Table

**Pull request merged**